src/components/Node/Node.tsx

```diff
@@ -54,9 +54,14 @@ const Node: FC<NodeProps> = ({
         <NodeIcon pathwayState={pathwayState} />
         {label}
       </div>
-      {isGuidanceState(pathwayState) && expanded && (
+      {expanded && (isGuidanceState(pathwayState) || documentation) && (
```

Comment:
Since this now renders for any node which has documentation it probably shouldn't be called RecNode anymore (since it is not just for recommendation nodes)

Reply:
Will rename to DocNode